src/pages/search/RegistrationSearch.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Typography } from '@material-ui/core';
+import useSearchRegistrations from '../../utils/hooks/useSearchRegistrations';
+import SearchResults from './SearchResults';
+import ListSkeleton from '../../components/ListSkeleton';
+
+interface RegistrationSearchProps {
+  searchTerm?: string;
+  noHitsText?: string;
+}
+
+const RegistrationSearch: FC<RegistrationSearchProps> = ({ searchTerm, noHitsText }) => {
+  const [registrationsSearch, isLoadingSearch] = useSearchRegistrations(searchTerm);
+  const { t } = useTranslation('common');
+
+  return (
+    <>
+      {isLoadingSearch ? (
+        <ListSkeleton arrayLength={3} minWidth={40} height={100} />
```

Comment:
litt usikker på hva jeg synes om skeletons i trefflister ....   Man aner jo ikke hvor lang trefflisten kommer til å være - så det blir ikke samsvar med skjelett og resultat. Tenker det funker best til innlasting av komponenter i en fast layout.

Reply:
godt poeng. vi har egentlig ikke lagt noen bestemt plan for hvordan vi skal vise lasting enda, så vi har litt spinner, og litt skeleton spredd. bør nok avklares om ikke veldig lenge